### ISSUE TITLE
Introductions

### ISSUE THREAD

**Issue by cheshire137** [cheshire137/overwatch-team-comps#2]
👋 Since so many folks want to help out on this project, I thought we could have some introductions to share how we'd each like to contribute. This should help with requesting code reviews on pull requests and knowing which work to tackle.

**Comment by cheshire137** [comment]
I'll start! 😀 I plan to do general development to implement features and test them. I'll also translate @RobThePM's vision into tasks anyone can pick up, assign themselves to, and work on. I also want to do code review, so please @-mention me on your pull requests! 😻

**Comment by ihavenotea** [comment]
I am happy to implement features and engage in code review.

DVa main and Battle Mercy. 😳

**Comment by pseudobeard** [comment]
Do you need CI pipelines? Cause that is what I build lol

**Comment by RobThePM** [comment]
Hi guys!

I’ll be coordinating the project and trying to establish some sense of process and organization on the non-development side. 😅 

To that end, I’ve posted a quick Project Guidelines summary: https://trello.com/c/d9pd5xFe.

There’s also a high level Project Plan: https://trello.com/c/ZAvFh5zU.

I will try to work on as many tasks as I can that are not related to development or design. 😀

**Comment by cheshire137** [comment]
> Do you need CI pipelines?

Yes! This app will have tests 🙏 and I'd like to see the results of those tests every time someone opens a pull request for review. My thinking was we'd use Travis CI to run tests and possibly also style checkers like Rubocop and ESLint. When we get some tests written and I open an issue about setting up Travis, feel free to snag that one, Sonny. 😀

**Comment by rewinfrey** [comment]
👋 my name is Rick but you likely know me as bashrw. 

I'm happy to feature develop, code review, and participate in planning and issue tracking.

I 💖 🍐 (pair programming) and I'm always happy to 🍐 .

Also please @-mention me on any issues you want to talk over. I'm happy to play the role of rubber 🦆!

Closest thing to a main for me Zen, but I have a special place in my heart dedicated to all heroes!

/cc @smbriones for possible design help and product design

**Comment by siege911** [comment]
Hey y'all. I'm CJ (siege911). To be honest, I'm probably one of the least experienced developers here and my time is extremely limited but I'm happy to help where I can. My experience in the past has been small solo projects as side jobs: small business websites, small mobile apps, etc. But my professional experience is all in a traditional IT shop. Networking. Server deployments. Automating IT business processes. Management. Etc. I'm currently an IT Manager over Networking, Systems and Telecom. So be nice if I seem like a noob when it comes to this type of development project. I've also only worked in .NET and Javascript.

Would probably like to focus a bit more on the beta testing and QC side. I do like trying to break things. :-)

**Comment by smbriones** [comment]
Hi, I'm Stephanie, aka drlngo. 👋 

Super excited to help out where needed on the design front! I'm a product designer, and some of my skills are things like UI and visual design, and implementation (HTML, CSS, and a tiny bit of JS).

I'm a D.Va main, (but also love Roadhog and Lucio lots). I have yet to play Orisa, however I feel like she may quickly become my new favorite. 😍 

Very happy to get to work with you all! ❤️

**Comment by summasmiff** [comment]
Hello! I'm Summer, aka summasmiff. 

I'm a ~~full stack~~ jack of all trades happy to help out with feature dev! Also, I can help out @smbriones with design-y stuff. 

D.Va / Symmetra main!

**Comment by justbarreto** [comment]
Whaaaat's up guys! I'm Emma, or as you may know me > Zion, ready to dish out some sass/css and some neat javascript transitions. I've worked with UI within the game development and Web Dev field. Not so much on the mobile. You'll usually find me extending classes or trying to find someway to minimize the repetition of css. I really like minimal/clean designs with a sense of introduction, where they welcome the user and slightly introduce the form of interaction. Outside of that I have fairly good illustration and graphic design skills. I just don't use them often :( .

I don't really have a main on Overwatch since I'm still fairly new at the game( 💯 truth +)

**Comment by RobThePM** [comment]
Hi everyone!

We’ve confirmed which features/requirements we want for the first version of this product, and they can be found [here](https://github.com/cheshire137/overwatch-team-comps) (scroll down to the README file). There’s also a link to the more detailed requirements document, as well as sample mockup screenshots! Hopefully this will give volunteers a better idea of what we’re trying to achieve!

Our goal with this first version is to release a product that is simple, but does a great job of achieving its desired functionality: allowing Overwatch players to save team compositions for all of the points and maps.

There were a lot of other really great ideas and we’ve archived those for now; hopefully we will be able to work on those in future versions!

If you are a developer or designer and would like to help contribute, go to [this page](https://github.com/cheshire137/overwatch-team-comps/issues) and feel free to start working on any of the issues.

If you have any questions regarding development, contact @cheshire137.
If you have any questions regarding design, contact @justbarreto.
For everything else, contact @RobThePM.

Thanks again to everyone so far!

**Comment by celluj34** [comment]
I'm late to the party but I would like to help if I can!

My name's Joe and I am a Software Developer by trade. I work in C#/SQL/ES6/gulp/Angular 1/knockout/jasmine/Specflow... I think that's all the major food groups. I hate the term "full-stack" developer but that's basically what I am. I prefer server-side but I'm getting out of my comfort zone more and more.

I've never coded a thing in Ruby (or Rails), but I'm willing to give it a shot.

If I had a main, it'd be Junkrat or Zarya.

**Comment by asg** [comment]
Ahh, better late than never. My name is Ami (no i'm not a girl and no it's not pronounced Amy), you may know as seed.

My current trade is tech evangelist pre-sales (_NJ housewife finds this one cool trick that will save you millions on your PaaS implementation!_). In previous lives i've been be a graphic designer, developer, solution arch, business analyst, project manager and cat herder. I know lots of languages, methodologies and stuff, but most of them are out of date apart from maybe my C#/Unity/VR/AR knowledge (which probably isn't much help here). 

Happy to help in any way, but i'm guessing i'll mostly be a tester/suggester of requirements.

I main zen, and in the great words of the omnic himself _'Wubba lubba dub dub'_.
